python/cudf/dataframe/index.py

```diff
@@ -218,6 +266,12 @@ def __new__(self, values, name=None):
         res.name = name
         return res
 
+    def copy(self, deep=True):
+        if(deep):
+            return deepcopy(self)
```

Comment:
I don't think there's a __copy__ or __deepcopy__ for Column, so wouldn't this not deepcopy the _values attribute?